apps/documentation/src/app/examples/button/button.tailwind.example.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+import { NgpButton } from 'ng-primitives/button';
+
+@Component({
+  selector: 'app-button-tailwind',
```

Comment:
I think I'm happy to remove the "tailwind" suffixes here, just to keep it simpler, and will make it easier to update the example generator to produce this tailwind file as well

```suggestion
  selector: 'app-button',
```